examples/scripts/countdown.js

```diff
@@ -80,7 +80,7 @@ WoT.produce({
                 const listToDelete = [];
                 for (const id of countdowns.keys()) {
                     const as = countdowns.get(id);
-                    if (as !== undefined && as.output !== undefined) {
+                    if ((as === null || as === void 0 ? void 0 : as.output) !== undefined) {
```

Comment:
Interesting to see what the tool-chain makes out of
`if (as?.output !== undefined) {`

Note: This is not a change in this PR.. we might have forgotten to update the JS in a previous PR..